datalad_catalog/webcatalog.py

```diff
@@ -37,7 +38,7 @@ class WebCatalog(object):
 
     # Get package-related paths
     package_path = Path(__file__).resolve().parent
-    config_dir = package_path / "config"
+    default_config_dir = package_path / "config"
```

Comment:
Note: using `importlib.resources` seems to be recommended over `__file__`; see explanation: [accessing data files at runtime](https://setuptools.pypa.io/en/latest/userguide/datafiles.html#accessing-data-files-at-runtime) in setuptools docs (also note that specific functions were backported under `importlib_resources` for python <3.9).

However, I'm no expert and only learned about this recently, so just sharing. `__file__` works perfectly fine except some exotic situations. So feel free to disregard or relegate to an issue for the future.

Reply:
Thanks for the pointer. I created a separate issue for this, since `__file__` is used in multiple places in the codebase.

https://github.com/datalad/datalad-catalog/issues/243